proto/cosmos/nft/v1beta1/query.proto

```diff
@@ -14,16 +14,31 @@ service Query {
     option (google.api.http).get = "/cosmos/nft/v1beta1/balance/{owner}/{class_id}";
   }
 
+  // BalancebyQueryString queries the number of NFTs of a given class owned by the owner, same as balanceOf in ERC721
+  rpc BalanceByQueryString(QueryBalanceByQueryStringRequest) returns (QueryBalanceByQueryStringResponse) {
```

Comment:
```suggestion
  //
  // Since: nft v0.1.1
  rpc BalanceByQueryString(QueryBalanceByQueryStringRequest) returns (QueryBalanceByQueryStringResponse) {
```